### ISSUE TITLE
Missing key(s) when loading model error when training pi0

### ISSUE THREAD

**Issue by sdfzz** [huggingface/lerobot#1733]
Hi, 

I'm trying to train pi0 on LIBERO, but I got the following error upon running train.py

Missing key(s) when loading model: ['normalize_inputs.buffer_observation_state.mean', 'normalize_inputs.buffer_observation_state.std' ...]

Has anyone encountered the same issue? Is there any fix on this?

Any help would be greatly appreciated

Regards,

Steve

**Comment by AdilZouitine** [comment]
Hey, we are aware of this issue. Usually, the workaround is to load the state_dict with `strict=False`, and then re-instantiate the normalization layer inside the policy.
FYI, once this PR is merged, this kind of issue will disappear: https://github.com/huggingface/lerobot/pull/1452

**Comment by sdfzz** [comment]
Hey, thank you for the reply.

I ended up manually checking the missing key values (print(self.normalize_inputs.buffer_observation_state.mean))

and then manually add them to transformed_state_dict (transformed_state_dict['normalize_inputs.buffer_observation_state.mean'] = torch.tensor([...]).cuda())

Hopefully this issue will be fixed quickly with PR #1452 

Regards,

Steve
